### PR TITLE
Fix WEBAPP-3040/WEBAPP-3078/WEBAPP-3306/WEBAPP-3417

### PR DESCRIPTION
**Pull request by herrmannplatz**

### CODE REVIEW

File: test/unit_tests/links/LinkPreviewBlackListSpec.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  it 'should return true if link is spotify', ->
+    url = 'spotify.com'
+    expect(z.links.LinkPreviewBlackList.is_blacklisted(url)).toBeTruthy()
+
+  it 'should return true if link is vimeo', ->
+    url = 'vimeo.com'
+    expect(z.links.LinkPreviewBlackList.is_blacklisted(url)).toBeTruthy()
+
+  it 'should return false if link is spotify', ->
```
Comment (lipis) [inline test/unit_tests/links/LinkPreviewBlackListSpec.coffee:21]: Spotify, Vimeo, YouTube :)

**Comment by herrmannplatz** [comment]
https://github.com/wireapp/wire-desktop/issues/84

File: test/unit_tests/links/LinkPreviewBlackListSpec.coffee (hunk 1 above)
Comment (herrmannplatz) [reply test/unit_tests/links/LinkPreviewBlackListSpec.coffee:21]: ?

Comment (gregor) [reply test/unit_tests/links/LinkPreviewBlackListSpec.coffee:21]: Guess Lipis is asking for alphabetical test order

Comment (lipis) [reply test/unit_tests/links/LinkPreviewBlackListSpec.coffee:21]: no sorry.. the case.. spotify > Spotify, etc..  but not important in any case .. :) missed the question `?`

**Pull request merged**